src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJobOptions.java

```diff
@@ -514,4 +521,23 @@ public List<SegmentToTranscribe> getSegmentsToTranscribe() {
   public void setSegmentsToTranscribe(List<SegmentToTranscribe> segmentsToTranscribe) {
     this.segmentsToTranscribe = segmentsToTranscribe;
   }
+
+   /**
+   * Returns the list of speaker names
+   *
+   * @return List of speaker names
+   */
+  public List<String> getSpeakerNames() {
+    return speakerNames;
+  }
+
+  /**
+   * Specifies the list of speaker names in an audio file
+   * This property is optional but can only be used with "human" transcriber.
```

Comment:
ditto as previous comment. I think we should keep optional in the description, but defer the explanation of human usage only to the API docs site. I don't feel too strongly on this point, so please push back if you think otherwise.